site/app/controllers/admin/PlagiarismController.php

```diff
@@ -575,6 +582,13 @@ public function ajaxGetSubmissionConcatenated($gradeable_id, $user_id_1, $versio
 
         $rankings = $this->getOverallRankings($gradeable_id);
 
+        if (count($rankings) === 0 || $rankings === null) {
```

Comment:
This condition needs to be flipped so that it's `if ($rankings === null || count($rankings) === 0) {` as  `count(null)` will throw a PHP warning.

Reply:
I will change this in the next PR.